lib/osf-components/addon/components/osf-navbar/preprint-links/template.hbs

```diff
@@ -20,7 +20,7 @@
 {{/if}}
 <li data-test-nav-submit-preprint-link>
     <OsfLink
-        @href='/preprints/submit/'
+        @href='/preprints/osf/submit/'
```

Comment:
As Brian mentioned, should this go to the select page on the non-branded (OSF) navbar?? Also, this can also be an in-app transition now, so this should use `@route` (and `@models` if needed)